diff --git a/dist/index.js b/dist/index.js
--- a/dist/index.js
+++ b/dist/index.js
@@ -152,7 +152,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-const API_VERSION = "v1";
+const API_VERSION = "v1beta";
 /**
  * We can't `require` package.json if this runs on web. We will use rollup to
  * swap in the version number here at build time.
diff --git a/dist/index.mjs b/dist/index.mjs
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -150,7 +150,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-const API_VERSION = "v1";
+const API_VERSION = "v1beta";
 /**
  * We can't `require` package.json if this runs on web. We will use rollup to
  * swap in the version number here at build time.
